src/views/IndividualTeamView.vue

```diff
@@ -101,23 +106,14 @@ export default {
   data() {
     return {
       team: {},
+      userTeamRole: '',
```

Comment:
Is there any benefit to separating this out from the team response object?

Reply:
If you want easy access to the field, maybe make a computed value that returns team.userTeamRole?

Reply:
Agreed